### DISCUSSION TITLE
Using renderFunction to change waveColor, but it will be overwritten by progressColor.

### DISCUSSION THREAD

**Discussion by sunyayue** [katspaugh/wavesurfer.js#3671] [Q&A] [answered]
Using renderFunction to change waveColor, but it will be overwritten by progressColor.

 Can we add a configuration to turn off progressColor and keep it in the style initialized by renderFunction

Or do I have other ways to achieve this goal?

![image](https://github.com/katspaugh/wavesurfer.js/assets/68684132/bed88a42-f6ba-4efb-96ea-4ae50e5e05eb)

**Comment by katspaugh** [comment, marked as answer]
You could set the `progressColor` to `transparent` and add this CSS to hide the overlay completely:

```
::part(canvases) {
  clip-path: none !important;
}
```

See an example of CSS styling here: https://wavesurfer.xyz/examples/?styling.js

**Reply by sunyayue** [reply]
Thanks a million
